src/components/CustomCheckbox/__tests__/CustomCheckbox.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import CustomCheckbox from '..';
+import { render, screen } from '@testing-library/react';
+
+const registerMock = jest.fn();
+
+describe('CustomCheckbox', () => {
```

Comment:
please provide more test cases for this component. 
- does it support passing label text? 
- will it render the label correctly?
- will it change the checked value of ]input when we click on the label?
- will it change the checked value of input when we click on the input?

Reply:
added test cases for checking the checkbox and label